src/System.Memory/src/System/SpanSortHelpers.Keys.cs

```diff
@@ -0,0 +1,234 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+#if !netstandard
+using Internal.Runtime.CompilerServices;
+#endif
+
+using static System.SpanSortHelpersCommon;
+using S = System.SpanSortHelpersKeys;
```

Comment:
Why do we need this using directive?

Reply:
Since Sort methods are identical in signature for many different variants and members to avoid recursive calls, this using statement make sure we call the static member, not some local member. I'll just update to call the static call directly without the using, to avoid any confusion.